robottelo/cli/hostcollection.py

```diff
@@ -101,6 +105,12 @@ def hosts(cls, options=None):
             cls._construct_command(options), output_format='csv')
 
     @classmethod
+    def list(cls, options=None, per_page=True):
+        """List host collections"""
+        cls.transform_ids(options)
```

Comment:
if you do not want to refactor the function transform_ids to check if options is None you have to check here